### PR TITLE
update nftables to 1.0.9

### PR DESCRIPTION
**Pull request by ddstreetmicrosoft**
Fixes: #7301

###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [ ] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [ ] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?
update nftables to 1.0.9

###### Does this affect the toolchain?  

no

###### Associated issues  

- #7301 

###### Test Methodology

- Pipeline build id: xxxx
https://dev.azure.com/mariner-org/mariner/_build/results?buildId=486288&view=results

### CODE REVIEW

File: SPECS/nftables/nftables.spec [hunk 1]
```diff
@@ -59,41 +62,39 @@ The nftables python module provides an interface to libnftables via ctypes.
...
-	--enable-python --with-python-bin=%{__python3}
+%configure --disable-silent-rules --with-xtables --with-json
 %make_build
+cd py/
+%pyproject_wheel
 
 %install
 %make_install
-find %{buildroot} -type f -name "*.la" -delete -print
+find $RPM_BUILD_ROOT -name '*.la' -exec rm -f {} ';'
```
Comment (reubeno) [inline SPECS/nftables/nftables.spec:72]: Interesting; is `RPM_BUILD_ROOT` env var preferred over `%{buildroot}` macro var?

File: SPECS/nftables/nftables.spec [hunk 2]
```diff
@@ -1,55 +1,58 @@
...
-BuildRequires:  iptables-devel
-BuildRequires:  jansson-devel
-BuildRequires:  libedit-devel
-BuildRequires:  libmnl-devel
-BuildRequires:  libnftnl-devel
-BuildRequires:  make
-BuildRequires:  python3-devel
-BuildRequires:  readline-devel
-BuildRequires:  systemd
+#BuildRequires: autogen
```
Comment (reubeno) [inline SPECS/nftables/nftables.spec:17]: Why are these commented out?

Comment (ddstreetmicrosoft) [reply SPECS/nftables/nftables.spec:17]: taken unmodified from fedora spec

File: SPECS/nftables/nftables.spec (hunk 1 above)
Comment (ddstreetmicrosoft) [reply SPECS/nftables/nftables.spec:72]: and they are equivalent AFAIK other than what stage they are evaluated (i.e. macro by rpm, and env var by bash)

**Pull request closed without merging**